concepticondata/conceptlists/Migliazza-1972-599.tsv

```diff
@@ -0,0 +1,715 @@
+	506	stone rock	2125	STONE OR ROCK
+	507	stop cease	1928	CEASE
+	508	straight	1404	STRAIGHT
+	509	strong	785	STRONG
+	510	suck inhale		
+	511	sun	1343	SUN
+	512	swallow	897	SWALLOW
+	513	sweep	230	SWEEP
+	514	sweet	717	SWEET
+	515	swell	1573	SWELL
+	516	swim	1439	SWIM
+	517	swing hammock		
+	518	tail	1220	TAIL
+	519	take pick up	371	PICK UP
+	520	tall	711	TALL
+	521	tapir	312	TAPIR
+	522	tasteless	1771	TASTELESS
+	523	teach show how	709	TEACH
+	524	tear V. rip	1735	TEAR (SHRED)
+	525	tell news		
+	526	there	1937	THERE
+	527	there near hearer		
+	528	thick	1244	THICK
+	529	thin	2308	THIN
+	530	thin animate		
+	531	thing it		
+	532	think	2271	THINK
+	533	thirsty	847	THIRSTY
+	534	thorn	124	THORN
+	535	throw	1456	THROW
+	536	throw away	3696	THROW AWAY
+	537	thunder	1150	THUNDER
+	538	thus	3413	IN THIS WAY
+	539	tickle	3647	TICKLE (SOMEBODY)
+	540	tie V.	1917	TIE
+	541	tight	3053	TIGHT
+	542	tire V.	1996	GET TIRED
+	543	tobacco	974	TOBACCO
+	544	today	1283	TODAY
+	545	tongue	1205	TONGUE
+	546	tooth	1380	TOOTH
+	547	toucan sp.	308	TOUCAN
+	548	touch V.	1892	TOUCH
+	549	trave		
+	550	treat V. cure	499	CURE
+	551	tree stick wood	2141	TREE OR WOOD
+	552	true	1657	TRUE
+	553	true word real		
+	554	turn	1588	TURN
+	555	turn around	1444	TURN AROUND
+	556	turn back		
+	557	turn over	3537	TURN OVER
+	558	twisted		
+	559	two	1498	TWO
+	560	umbilicus	1838	NAVEL
+	561	under below	1485	BELOW OR UNDER
+	562	understand	1536	UNDERSTAND
+	563	untie	352	UNTIE
+	564	unwilling		
+	565	upright	1442	STAND
+	566	upstream		
+	567	urine	1142	URINE
+	571	village	930	VILLAGE
+	572	vine sp.	343	VINE
+	573	visit	1377	VISIT
+	574	vomit	1278	VOMIT
+	575	wait (later)	1590	WAIT (FOR)
+	576	wake up	1602	WAKE UP
+	577	walk	1443	WALK
+	578	want desire	2117	DESIRE
+	579	want sex		
+	580	warm	1232	WARM
+	581	wash	1453	WASH
+	582	water	948	WATER
+	583	weak	1601	WEAK
+	584	wet	1726	WET
+	585	what	1236	WHAT
+	586	when	1238	WHEN
+	587	where	1237	WHERE
+	588	which	727	WHICH
+	589	white	1335	WHITE
+	590	who	1235	WHO
+	591	why	1270	WHY
+	592	wide	1243	WIDE
+	593	wind	960	WIND
+	594	wing	1257	WING
+	595	wipe	1454	WIPE
+	596	with	1340	WITH
+	597	woman female	962	WOMAN
+	598	work	984	WORK (LABOUR)
+	599	worm sp.	1219	WORM
+	600	yellow	1424	YELLOW
+	601	yawn	1028	YAWN
+	604	young	1207	YOUNG
+	605	young man teen	76	YOUNG MAN
+	606	young woman	77	YOUNG WOMAN
+	607	Fa bro	1985	PATERNAL UNCLE (FATHER'S BROTHER)
+	608	Mo MoSi		
+	609	Son	1620	SON
+	610	Da.	1357	DAUGHTER
+	611	Bro. Older	1759	OLDER BORTHER
+	612	Si Older	1758	OLDER SISTER
+	613	Sibling Younger (same sex)	3038	SAME-SEX YOUNGER SIBLING
+	614	Sibling Younger (opposite sex)	3040	DIFFERENT-SEX YOUNGER SIBLING
+	615	Da Son Si Son		
+	616	Si Da Da Da		
+	617	Mo Bro Mo Fa Wi Fa		
+	618	Fa Si Fa Mo Wi Mo		
+	620	Wi	1199	WIFE
+	621	Si Hu Wi Bro		
+	108-110	come	1446	COME
+	13-14	all	98	ALL
+	165-166	enemy	1358	ENEMY
+	226-227	good	1035	GOOD
+	24-25	ant sp.	587	ANT
+	305-306	like V.	1117	LIKE
+	316-317	loose	2506	LOOSE
+	330-331-332	many	1198	MANY
```

Comment:
Please separate multiple ID's by coma, and sort them correctly

Reply:
what do you mean with shorting them correctly?

Reply:
Well, ID 24 should be close to row 24, I guess, right? So that we have the same order of concepts in the list here as in the original. Of course with slightly different IDs, but at least the same order

Reply:
oh, I misread then! well, now it is like that. first the general vocabulary list and them the ones in the appendix.